refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/RequiresComputationTest.java

```diff
@@ -8,54 +8,67 @@
 import com.sun.source.tree.ReturnTree;
 import org.junit.jupiter.api.Test;
 
-final class IsLikelyTrivialComputationTest {
+final class RequiresComputationTest {
   @Test
   void matches() {
     CompilationTestHelper.newInstance(MatcherTestChecker.class, getClass())
         .addSourceLines(
             "A.java",
+            "import java.io.OutputStream;",
             "import java.util.function.Predicate;",
             "",
             "class A {",
-            "  String negative1() {",
-            "    return String.valueOf(1);",
+            "  int negative1() {",
+            "    int[] arr = new int[0];",
+            "    return arr[0];",
             "  }",
             "",
             "  String negative2() {",
-            "    return toString().toString();",
+            "    return null;",
             "  }",
             "",
-            "  String negative3() {",
-            "    return \"foo\" + toString();",
+            "  boolean negative3() {",
+            "    return false;",
             "  }",
             "",
-            "  byte negative4() {",
-            "    return \"foo\".getBytes()[0];",
+            "  int negative4() {",
+            "    return 0;",
             "  }",
             "",
-            "  int negative5() {",
-            "    int[] arr = new int[0];",
-            "    return arr[hashCode()];",
+            "  String negative5() {",
+            "    return \"foo\" + \"bar\";",
             "  }",
             "",
-            "  int negative6() {",
-            "    return 1 * 2;",
+            "  Predicate<String> negative6() {",
+            "    return v -> \"foo\".equals(v);",
             "  }",
             "",
-            "  Predicate<String> negative7() {",
-            "    return toString()::equals;",
+            "  A negative7() {",
+            "    return this;",
             "  }",
             "",
-            "  String negative8() {",
-            "    return (toString());",
+            "  Predicate<String> negative8() {",
+            "    return \"foo\"::equals;",
             "  }",
             "",
-            "  Object negative9() {",
-            "    return (Object) toString();",
+            "  OutputStream negative9() {",
+            "    return System.out;",
```

Comment:
This new test is to cover existing code that was previously untested. I filed hcoles/pitest#1341.